wordpress-mobile/WordPress-iOS
# Share extension: Associate Post and uploaded Media items

Fixes #9070 

Associate Media items with Post after a post has been successfully created from the share extension.

To test:
1. Create a post from the Share extension. Open Photos, select an image, share it to WordPress, and compose a Post
2. Navigate in a browser to the blog's media library and check that the image that was just uploaded is associated to the post that was just created.

In my case, for a self-hosted blog, it looks something like this:
<img width="417" alt="screen shot 2018-06-05 at 1 33 06 pm" src="https://user-images.githubusercontent.com/2722505/41001760-bbb87a3e-68c6-11e8-99c4-4f76be24a90f.png">
